buildroot/share/tests/megaatmega2560-tests

```diff
@@ -324,5 +324,28 @@ exec_test $1 $2 "Mixed TMC configuration"
 #
 #
 
+#
+#test power supply
+#
+#
+restore_configs
+opt_enable POWER_SUPPLY_CONTROL_FEATURE
+opt_set POWER_SUPPLY_PIN_INVERTING true
+opt_enable PS_DEFAULT_OFF
+opt_enable AUTO_POWER_CONTROL
+opt_enable AUTO_POWER_E_TEMP
+opt_enable REPRAP_DISCOUNT_SMART_CONTROLLER SDSUPPORT
+exec_test $1 $2 "Power Supply configuration 1"
+
+restore_configs
+opt_enable POWER_SUPPLY_CONTROL_FEATURE
+opt_set POWER_SUPPLY_PIN_INVERTING false
+opt_enable PS_DEFAULT_OFF
+opt_enable AUTO_POWER_CONTROL
+opt_enable AUTO_POWER_E_TEMP
+opt_enable REPRAP_DISCOUNT_SMART_CONTROLLER SDSUPPORT
+exec_test $1 $2 "Power Supply configuration 2"
```

Comment:
Good to have new tests, but it is always better if they can be added to an existing test. And, when choosing which platform to add to, pick the one with the shortest time to completion (if possible). This will help to keep the test times down to a minimum. We might consider splitting them into more jobs within each platform if the duration continues to be very long. As it is now, all the above run serially.

Reply:
You are right, if the test would be ought to stay. These are specifically made to test my changes in this PR. The commit text for the test states: "Revert me before merge."

Still developing the workflow for me. Local Travis tests would be perfect for me. This time i wanted to send this PR only to my own repository until in a good shape, what i missed. Otherwise i'd tried to force push not that often.

Reply:
Next time i'll try to make a additional speciffic-test-branch based on a remove-general-tests-branch, where i can simply delete most of the normally running tests. Touching every time half a dozen of files is too boring to do it more than a few times.